gravitee-apim-e2e/api-test/src/policies/mock-policy.spec.ts

```diff
@@ -92,11 +92,13 @@ describe('Mock policy', () => {
       );
       await succeed(apiManagementApiAsApiUser.deployApiRaw({ orgId, envId, api: createdApi.id }));
 
-      await fetchGatewaySuccess({ contextPath: createdApi.context_path })
-        .then((res) => res.json())
-        .then((body) => {
-          expect(body).toEqual(JSON.parse(mockContent));
-        });
+      await fetchGatewaySuccess({
+        contextPath: createdApi.context_path,
+        expectedResponseValidator: async (response) => {
```

Comment:
This test made an api redeployment which can take up to 6s to be taken into account by the GW.   
and meanwhile the previously deployed api is still present. 
which can cause this kind of error :
<img width="1046" alt="image" src="https://user-images.githubusercontent.com/4974420/177154812-76e199e4-063a-404e-8314-4f098b65c833.png">

with `expectedResponseValidator` the retry can be triggered by other properties than statues